### ISSUE TITLE
Instead of using access groups, use a whitelist of netids

### ISSUE THREAD

**Issue by tnip** [PrairieLearn/PrairieLearn#170]

**Comment by mwest1066** [comment]
You mean instead of `"role": "TA"`, you want something like `"uids": ["mwest@illinois.edu", "nip2@illinois.edu"]`?

If so, see the recent change in 1.14.0. I think I haven't pushed this for 421 yet?

**Comment by tnip** [comment]
Well, I'd like something like:

```
[
  {
    "role": "TA",
    "mode": "Public"
  },
  {
    "role": "Student"
    "whitelist": ["mwest@illinois.edu", "nip2@illinois.edu"],
    "mode": "Public"
  }
]

Which should translate to:

All listed TAs should have public access, and people in the whitelist should also have public access (as a student). I'm not sure how I feel about adding role:Student since the TA/Instructor roles are preset from the get-go (i.e., by default, whitelisted people should be students).

**Comment by mwest1066** [comment]
Dup of #155, implemented in 72110a44a9b2c61eb435947a138b298884e0e813
